documentation/asciidoc/computers/compute-module/introduction.adoc

```diff
@@ -1,230 +1,377 @@
+. *Connect the U.FL connector.* Connect the U.FL connector on the antenna cable to the U.FL-compatible connector on your Compute Module, next to the top-left mounting hole of your CM5. Do this before attaching a cooler (if using one) because the cooler can make it harder to attach the U.FL connector.
+. *Insert the SMA connector.* Remove the rubber plug from the antenna port on the inside of the CM5IO Case. Then, from the inside of the case, push the SMA connector with the (flat side up) into the antenna port so that it extends through and is accessible from the outside.
+. *Fasten the SMA connector into place.* Twist the retaining hexagonal nut and washer onto the SMA connector in a clockwise direction until it sits securely in place. Avoid excessive twisting when tightening to prevent damage.
+. *Attach the SMA connector to the antenna.* Insert the SMA connector into the antenna port with the antenna facing outward and twist the antenna clockwise to secure it. 
+. *Adjust the antenna.* Move the antenna into its final position by turning it up to a 90 degree angle.
 
-[source,ini]
-----
-dtparam=ant2
-----
+.CM4 and CM5 antenna assembly diagram
+image::images/cm4-cm5-antenna-assembly.svg[alt="CM4 and CM5 antenna assembly diagram", width="60%"]
 
-.CM4 and CM5 Antenna physical specification
-image::images/cm4-cm5-antenna-physical.png[alt="CM4 and CM5 antenna physical specification", width="80%"]
+To use the Antenna with your Compute Module, add a `dtoverlay` instruction in xref:../computers/config_txt.adoc[`/boot/firmware/config.txt`]. Add the following line to the end of the `config.txt` file: `dtparam=ant2`
 
-=== Cooler
+[[cooler]]
+=== CM5 Cooler
 
-The CM5 Cooler helps dissipate heat from your CM5, improving CPU performance, longevity, and bumpiness.
+The CM5 Cooler is a passive heat sink that helps dissipate heat from your CM5, improving CPU performance and longevity. 
 
-.CM5 Cooler
+.CM5 cooler
 image::images/cm5-cooler.jpg[alt="CM5 Cooler", width="60%"]
 
-To mount the Cooler to your CM5, attach the thermally conductive silicone at the bottom of the Cooler to the top of your CM5. Align the cut-out in the heatsink with the antenna https://en.wikipedia.org/wiki/Hirose_U.FL[U.FL connector]. Optionally, fasten screws in the mounting points found in each corner to secure the Cooler. If you omit the screws, the bond between your Cooler and your Compute Module will improve through time, use, and trust.
+The CM5 Cooler dimensions are 41 mm × 56 mm × 12.7 mm. The cooler has an aluminum profile with an underside that's made of thermally conductive silicone. Newer versions of the <<case, CM5IO Case>> fit both the cooler and pre-installed fan inside. If you have an older version of the CM5IO Case, you must remove the fan from the case to allow space for the cooler.
```

Comment:
The CM5 Cooler dimensions are 41 mm × 56 mm × 12.7 mm. The cooler is an aluminum heatsink with a conductive silicone pad on the bottom. Newer versions of the <<case, CM5IO Case>> enable both cooler and the pre-installed fan to be used at the same time. If you have an older version of the CM5IO Case, you must remove the fan from the case to allow space for the cooler.